### PR TITLE
Fix lobby time input

### PR DESCRIPTION
**Pull request by PVince81**
Fixes https://github.com/nextcloud/spreed/issues/4983

- Bug: Lobby timer can now be cleared. When doing so it is saved directly.
- UX: Adjusted minute selection to have an interval of 5.
- UX: Adjusted default date to be the start of the next hour.
- UX: Removed extra click for confirmation as we anyway have the save button.
- Internal: Changed the data type for communicating with date picker to use timestamps, since this is what we store anyway.

### CODE REVIEW

File: src/components/ConversationSettings/LobbySettings.vue [hunk 1]
```diff
@@ -54,6 +54,7 @@
 							id="moderation_settings_lobby_timer_field"
 							aria-describedby="moderation_settings_lobby_timer_hint"
 							:value="lobbyTimer"
+							:defaultValue="defaultLobbyTimer"
```
Comment (nickvergessen) [inline src/components/ConversationSettings/LobbySettings.vue:57]: Attribute ':defaultValue' must be hyphenated

```suggestion
							:default-value="defaultLobbyTimer"
```

**Comment by wiswedel** [comment]
Thanks for taking care, @PVince81. I'm not qualified to approve or equipped to test the code, so I'll leave that to @nickvergessen.

**Comment by nickvergessen** [comment]
- [ ] Time selector does not close anymore when selecting a minute option
- [ ] When you select a time and click "anywhere" (empty space in the conversation settings modal) the popup will close, but you don't know whether you have to press the arrow button or not. It looks the same like when you close the time picker with clicking on the arrow

**Comment by nickvergessen** [comment]
I don't think it's related, but:
* When the conversation settings open you can scroll the complete window on the background
* When you trigger a notification scrolling goes totally wild
* After a view seconds of the notification all is fixed again
Peek 2021-01-21 16-18

**Comment by PVince81** [comment]
When the date picker was in the menu, you had to click on an "OK" button (confirm: true), and that would trigger a save.

Now that we have a separate save button, you have to click that to close the popup.
Otherwise closing the popup when selecting the minute is not a good idea as the person selecting the date/time might not be done

If all this is confusing I could remove the save button and bring the back the "OK in popup for saving". When clearing it anyway saves automatically now. Would that help to improve UX ?

**Comment by nickvergessen** [comment]
> Otherwise closing the popup when selecting the minute is not a good idea as the person selecting the date/time might not be done

I ment we should close the date/time picker when you click on a minute and then make the previous (should be) disable save button non-disabled and primary (to make it obvious you need to save now).

But since this is so weird, I would prefer to just have the OK button on the time-picker again. Felt much more natural and well after selecting the time, you are done... there is mostly nothing you want to do more in the date-time selector

**Comment by PVince81** [comment]
> But since this is so weird, I would prefer to just have the OK button on the time-picker again. Felt much more natural and well after selecting the time, you are done... there is mostly nothing you want to do more in the date-time selector

Yeah, let's try that

**Comment by PVince81** [comment]
I don't see any scrollbars in regular mode. Probably the dyslexia font that is pushing stuff further than expected, same with https://github.com/nextcloud/spreed/issues/4993

**Comment by PVince81** [comment]
@nickvergessen please re-review, now you need to click "Ok" in the popup for saving.

**Comment by PVince81** [comment]
@nickvergessen I tried dark mode + dyslexia font with various window widths but could not reproduce the scrolling problem when the notification is visible. Would need further research separately to find out what other conditions are involved.

**Review by nickvergessen — APPROVED** [review]

**Comment by nickvergessen** [comment]
Behaviour is a bit weird when another moderator changes things in parallel.
But okay for now

**Pull request merged**